contribute/style-guides/documentation-style-guide.md

```diff
@@ -85,19 +85,19 @@ The following guidelines are specific to Grafana documentation. For the most par
 - Task topic headings start with a verb.
   - Write a query
   - Create a dashboard
-- Concept and reference topic headings should be nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide
+- Concept and reference topic headings are nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide
```

Comment:
The sentence should stay as is.

Reply:
Please use suggestions so it is easier to apply the change.